base/abstractarray.jl

```diff
@@ -3,9 +3,10 @@
 ## Basic functions ##
 
 """
-    AbstractArray{T, N}
+    AbstractArray{T,N}
 
-Abstract array supertype which arrays inherit from.
+Abstract `N`-dimensional array supertype with elements of type `T`
+which arrays inherit from.
```

Comment:
I was the one adding the apparently awkward wording in #22052 so might as well update it.